src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/ProgressRing/ProgressRingTests.cs

```diff
@@ -13,7 +13,6 @@ public class ProgressRingTests
 	[RunsOnUIThread]
 	[DataRow(true)]
 	[DataRow(false)]
-	[RunsOnUIThread]
```

Comment:
This is needed, is there a reason to remove it?

Reply:
@jeromelaban It exists on line 13.

Looks like it was added in `master` and in breaking-changes separately on different lines. So the merge caused a duplicate attribute error.